.github/workflows/detect-changes.yml

```
@@ -9,13 +9,16 @@ on:
     outputs:
       dotnet:
         value: ${{ jobs.changes.outputs.dotnet }}
+      packages:
```

```suggestion
      nuget-packages:
```

maybe? to not confuse with python packages